financial-connections/src/main/java/com/stripe/android/financialconnections/features/notice/NoticeSheet.kt

```diff
@@ -77,6 +77,16 @@ private fun NoticeSheetContent(
                 onPrimaryButtonClick = onConfirmModalClick,
                 onSecondaryButtonClick = {} // TODO handle secondary button clicks.
             )
+            /**
+             * We're not expecting update required content on the generic notice sheet yet, as it's managed by
+             * [com.stripe.android.financialconnections.features.accountupdate.AccountUpdateRequiredModalKt].
+             */
+            is NoticeSheetContent.UpdateRequired -> GenericBottomSheetContent(
```

Comment:
Good for now. Let’s make sure to follow-up with the necessary work.

Reply:
https://jira.corp.stripe.com/browse/BANKCON-11725